### PR TITLE
bumping up the minimum agent version for tasks that depend on Deploym…

### PR DESCRIPTION
**Pull request by KrishnaAdityaB**
bumping up the minimum agent version for tasks that depend on Deployment Output Variable changes in agent

### CODE REVIEW

**Comment by KrishnaAdityaB** [comment]
@pavanadepu2 @kmkumaran @nigurr @prawalagarwal

File: Tasks/DeployAzureResourceGroup/task.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "demands": [
         "azureps"
     ],
-    "minimumAgentVersion": "1.95.1",
+    "minimumAgentVersion": "1.95.2",
     "groups": [
```
Comment (pavanadepu2) [inline Tasks/DeployAzureResourceGroup/task.json]: Isn't it requires update to task version

Comment (KrishnaAdityaB) [inline Tasks/DeployAzureResourceGroup/task.json]: udpated

**Comment by pavanadepu2** [comment]
looks fine

**Pull request merged**